check-compatibility.js

```diff
@@ -12,7 +12,7 @@ if (!knownPrototypeKitNames.includes(consumerPackageJson.name)) {
 }
 
 const compatibility = {
-  '1.22': {
+  '1.23': {
     'prototype-kit': ['9.8', '9.7', '9.6', '9.5', '9.4', '9.3', '9.2', '9.1', '9.0', '9.9', '9.10']
```

Comment:
The prototype kit if on version 9.11.2, could you add relevant versions

Reply:
Thanks, updated now.